include/ck/tensor_operation/gpu/device/device_base.hpp

```diff
@@ -42,6 +42,8 @@ struct BaseOperator
 
     virtual size_t GetWorkSpaceSize(const BaseArgument*) const { return 0; }
 
+    virtual void SetWorkSpacePointer(BaseArgument*, void*) const {}
```

Comment:
Should we just add a ```p_workspace_``` member function to ```BaseArgument``` and implement this function as below?

```
void SetWorkSpacePointer(BaseArgument*, void* p_workspace) const
{ BaseArgument->p_workspace_ = p_workspace; }
```

I can do it in my PR.